pcsx2/SPU2/spu2.cpp

```diff
@@ -50,20 +50,6 @@ u32 lClocks = 0;
 static bool CheckSSE()
```

Comment:
Why not just delete this function, the only usage is in the one below this.

Reply:
done, thanks for the catch, I thought it was a remnant of the plugin API but it turns out I deleted all traces of it!